### ISSUE TITLE
IndexOutOfBoundsException when attempting UploadEvent

### ISSUE THREAD

**Issue by GrahamBorland** [intercom/intercom-android#366]
## Version info
  - Intercom for Android version: 3.2.5
  - Android OS version: 6.0.1
  - Android Device: Nexus 5

## Expected behavior

Should not crash.

## Actual behavior

```
Fatal Exception: java.lang.RuntimeException: Could not dispatch event: class io.intercom.android.sdk.models.events.UploadEvent to handler [EventHandler public void io.intercom.android.sdk.conversation.NativeConversationContentPresenter.uploadSuccess(io.intercom.android.sdk.models.events.UploadEvent)]: Invalid index 0, size is 0
       at io.intercom.com.squareup.otto.Bus.throwRuntimeException(Bus.java:460)
       at io.intercom.com.squareup.otto.Bus.dispatch(Bus.java:387)
       at io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents(Bus.java:368)
       at io.intercom.com.squareup.otto.Bus.post(Bus.java:337)
       at io.intercom.android.sdk.MainThreadBus.access$001(MainThreadBus.java:9)
       at io.intercom.android.sdk.MainThreadBus$1.run(MainThreadBus.java:23)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5417)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by java.lang.IndexOutOfBoundsException: Invalid index 0, size is 0
       at java.util.ArrayList.throwIndexOutOfBoundsException(ArrayList.java:255)
       at java.util.ArrayList.get(ArrayList.java:308)
       at io.intercom.android.sdk.conversation.NativeConversationContentPresenter.createBlocks(NativeConversationContentPresenter.java:565)
       at io.intercom.android.sdk.conversation.NativeConversationContentPresenter.uploadSuccess(NativeConversationContentPresenter.java:548)
       at java.lang.reflect.Method.invoke(Method.java)
       at io.intercom.com.squareup.otto.EventHandler.handleEvent(EventHandler.java:89)
       at io.intercom.com.squareup.otto.Bus.dispatch(Bus.java:385)
       at io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents(Bus.java:368)
       at io.intercom.com.squareup.otto.Bus.post(Bus.java:337)
       at io.intercom.android.sdk.MainThreadBus.access$001(MainThreadBus.java:9)
       at io.intercom.android.sdk.MainThreadBus$1.run(MainThreadBus.java:23)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5417)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

## Steps to reproduce

Unknown. This is a field report from an end user.

## Logs

**Comment by ronocod** [comment]
We're looking into this now, will let you know when we have an update.

**Comment by henrikra** [comment]
Device:
samsung SM-G928F 
os: 6.0.1
We got something similar:
```
java.lang.RuntimeException Could not dispatch event: class io.intercom.android.sdk.models.events.UploadEvent to handler [EventHandler public void io.intercom.android.sdk.conversation.NativeConversationContentPresenter.uploadSuccess(io.intercom.android.sdk.models.events.UploadEvent)]: Invalid index 0, size is 0 
    Bus.java:460 io.intercom.com.squareup.otto.Bus.throwRuntimeException
    Bus.java:387 io.intercom.com.squareup.otto.Bus.dispatch
    Bus.java:368 io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents
    Bus.java:337 io.intercom.com.squareup.otto.Bus.post
    MainThreadBus.java:9 io.intercom.android.sdk.MainThreadBus.access$001
    MainThreadBus.java:23 io.intercom.android.sdk.MainThreadBus$1.run
    Handler.java:739 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    Looper.java:158 android.os.Looper.loop
    ActivityThread.java:7229 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:1230 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:1120 com.android.internal.os.ZygoteInit.main


Caused: java.lang.IndexOutOfBoundsException Invalid index 0, size is 0 
    ArrayList.java:255 java.util.ArrayList.throwIndexOutOfBoundsException
    ArrayList.java:308 java.util.ArrayList.get
    NativeConversationContentPresenter.java:551 io.intercom.android.sdk.conversation.NativeConversationContentPresenter.createBlocks
    NativeConversationContentPresenter.java:538 io.intercom.android.sdk.conversation.NativeConversationContentPresenter.uploadSuccess
    Method.java:-2 java.lang.reflect.Method.invoke
    EventHandler.java:89 io.intercom.com.squareup.otto.EventHandler.handleEvent
    Bus.java:385 io.intercom.com.squareup.otto.Bus.dispatch
    Bus.java:368 io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents
    Bus.java:337 io.intercom.com.squareup.otto.Bus.post
    MainThreadBus.java:9 io.intercom.android.sdk.MainThreadBus.access$001
    MainThreadBus.java:23 io.intercom.android.sdk.MainThreadBus$1.run
    Handler.java:739 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    Looper.java:158 android.os.Looper.loop
    ActivityThread.java:7229 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:1230 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:1120 com.android.internal.os.ZygoteInit.main
```

**Comment by ronocod** [comment]
Fixed in 3.2.7
